Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -46,6 +46,12 @@
           "ContentsRegex": "$SEC101/006.GitHubPat",
           "MessageArguments": { "secretKind": "GitHub personal access token (PAT)" }
```

Comment:
>GitHub personal access token (PAT)" [](start = 47, length = 35)

change this to 'GitHub personal access token (legacy format)

Reply:
Done.

---
In reply to: [608157641](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608157641) [](ancestors = 608157641)